--- goldwater-0.3.4/netengine/xlapi.c.orig	Sat Mar 29 13:21:11 2003
+++ goldwater-0.3.4/netengine/xlapi.c	Tue Oct 14 10:50:53 2003
@@ -27,6 +27,7 @@
  * -----------------------------------------------------------------------
  */
 #include <sys/types.h>
+#include <sys/select.h>
 #include <sys/socket.h>
 #include <signal.h>
 #include <stdio.h>
--- goldwater-0.3.4/tools/compileconf.c.orig	Sat Mar 29 13:21:11 2003
+++ goldwater-0.3.4/tools/compileconf.c	Tue Oct 14 10:55:30 2003
@@ -255,7 +255,7 @@
 int
 fn_import_xml_config( const char *fname )
 {
-  caddr_t mem     = NULL;
+  void   *mem     = NULL;
   struct  stat sb = {0};
   int     fd      = -1;
   int     ret     = 0;
@@ -274,8 +274,8 @@
   if( ret == 0 && fstat( fd, &sb ) < 0 ) ret = (-2);
 
   if( ret == 0 ) {
-    if( (mem = mmap( (caddr_t)0, (int)sb.st_size,
-		     PROT_READ, MAP_PRIVATE, fd, 0 )) >= (caddr_t)0 ) {
+    if( (mem = mmap( NULL, (int)sb.st_size,
+		     PROT_READ, MAP_PRIVATE, fd, 0 )) >= (void *)0 ) {
       WrapCStringPtrLen( &xmlCS, (char*)mem, (int)sb.st_size );
     } else ret = (-1);
   }
