--- goldwater-1.4.0/admin/Makefile.in~	Sat May 31 14:34:15 2003
+++ goldwater-1.4.0/admin/Makefile.in	Fri Dec  5 18:39:49 2003
@@ -32,7 +32,7 @@
 ALLINC  = -I. -I.. -I../gwlib -I@PXLBASE@/include
 CFLAGS = @CFLAGS@ -D_SVID_SOURCE @LOCAL_CFLAGS@ -DBUILDTARGET='"@BUILDTARGET@"' -c $(ALLINC)
 
-LFLAG   = -L../gwlib -L@PXLBASE@/lib -L/lib @LOCAL_LDFLAGS@ -lgwtx -lpxtra -lreadline -ltermcap  
+LFLAG   = -L../gwlib -L@PXLBASE@/lib -L/lib @LOCAL_LDFLAGS@ -lgwtx -lpxtra -lreadline
 
 OBJS = $(addprefix $(OBJ)/,$(OBJF))
 
